package.json

```diff
@@ -18,7 +18,7 @@
     "src"
   ],
   "scripts": {
-    "build": "ncc build src/index.mjs -m",
+    "build": "ncc build src/index.mjs --license upstream_licenses.txt",
```

Comment:
@nachoaIvarez With this option

Reply:
Cool. I thought that was just pointing to it not also creating one. Thanks.